packages/components/src/device-switcher/fixed-viewport.tsx

```diff
@@ -11,16 +12,28 @@ const deviceWidthById = {
 	phone: DEVICE_PHONE_WIDTH,
 } as Record< string, number >;
 
-export const useViewportScale = ( device: string, viewportWidth: number ) => {
-	const deviceWidth: number = deviceWidthById[ device ];
+const getDeviceWidth = ( device: string, previousDevice: string ) => {
+	if ( DEVICE_TYPES.GLOBAL_VIEW === device && deviceWidthById[ previousDevice ] ) {
+		return deviceWidthById[ previousDevice ] / 2;
```

Comment:
Why do we want to device it by 2 🤔

Reply:
It's not needed, I forgot to remove it 🫠